core/templates/dev/head/css/oppia.css

```diff
@@ -3928,10 +3928,17 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-activity-summary-tile .objective {
-  height: 60px;
+  height: 70px;
+  line-height: 1.35;
   font-family: "Rubik", "Roboto", Arial, sans-serif;
   font-size: 0.85em;
   padding: 12px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3; /* Truncating to 3 lines*/
+  max-height: 1.35*3; /* For non-webkit browsers
```

Comment:
(Although, obviously this will change based on the line-height you end up going with based on @markhalpin's comment.)